### ISSUE TITLE
How to get the MAP of the tained model? AND what does the total_loss mean 

### ISSUE THREAD

**Issue by sxyxf66** [morpheusthewhite/Faster-RCNN-TensorFlow-Python3.5#1]
Thanks for your work and share. It helps me a lot. Do you know how to get the MAP of the tained model? And what does the total_loss mean? My  total_loss painting is strange. One epoch poin has two tatal_loss values in the beginning. I don‘t know why it is. Hope your reply. Thanks .
<img width="427" alt="8888" src="https://user-images.githubusercontent.com/27723515/59599449-1e57b400-9131-11e9-9262-c1a042dfaf61.png">

**Comment by morpheusthewhite** [comment]
The mAP is not showed by default in tensorboard, you should calculate it (https://medium.com/@jonathan_hui/map-mean-average-precision-for-object-detection-45c121a31173).

The total loss is calculated as (in `lib/nets/network.py`, line 254)
```python
            loss = cross_entropy + loss_box + rpn_cross_entropy + rpn_loss_box
            self._losses['total_loss'] = loss
```

You have more than one point in the same graph because you didn't clear the tensorboard folder before starting a new run (you should do it every time otherwise previous data will remain there and will be shown)

**Comment by sxyxf66** [comment]
Thanks for your help. I don't know why I cannot open the link. Thanks a lot.

**Comment by morpheusthewhite** [comment]
You're welcome
